libpod/container_internal.go

```diff
@@ -134,6 +134,11 @@ func (c *Container) CheckpointPath() string {
 	return filepath.Join(c.bundlePath(), "checkpoint")
 }
 
+// CheckpointPath returns the path to the directory containing the checkpoint
```

Comment:
This function is called `PreCheckPointPath` not `CheckpointPath` - comment needs adjusting